tests/app/views/test_buyers.py

```diff
@@ -1874,6 +1882,396 @@ def test_404_if_lot_does_not_allow_brief(self, data_api_client):
+        brief = mock.Mock()
+
+        with po(buyers, 'get_sorted_responses_for_brief') as m:
+            result = self.instance.get_responses(brief)
+
+        assert result == m.return_value
+
+        m.assert_called_once_with(brief, self.instance.data_api_client)
+
+    def test_get_question(self):
+        po = functools.partial(mock.patch.object, autospec=True)
+
+        framework_slug = mock.Mock()
+        lot_slug = mock.Mock()
+        manifest = mock.Mock()
+
+        obj = self.content_loader.get_manifest.return_value
+        content = obj.filter.return_value
+
+        result = self.instance.get_questions(framework_slug, lot_slug, manifest)
+
+        assert result == content.get_section.return_value.questions
+
+        self.content_loader.get_manifest\
+            .assert_called_once_with(framework_slug, manifest)
+
+        obj.filter.assert_called_once_with({'lot': lot_slug})
+
+        content.get_section\
+               .assert_called_once_with('view-response-to-requirements')
+
+    def test_get_question_fails_with_empty_list(self):
+        po = functools.partial(mock.patch.object, autospec=True)
+
+        framework_slug = mock.Mock()
+        lot_slug = mock.Mock()
+        manifest = mock.Mock()
+
+        self.content_loader.get_manifest.return_value\
+                           .filter.return_value\
+                           .get_section.return_value = None
+
+        result = self.instance.get_questions(framework_slug, lot_slug, manifest)
+
+        assert result == []
+
+    def test_get_context_data(self):
+        po = functools.partial(mock.patch.object, autospec=True)
+
+        self.instance.get_responses = mock.Mock()
+
+        brief = api_stubs.brief(status='closed')
+
+        kwargs = {
+            'brief_id': mock.Mock(),
+            'framework_slug': mock.Mock(),
+            'lot_slug': mock.Mock()
+        }
+
+        expected = dict(**kwargs)
+        expected['brief'] = brief['briefs']
+        expected['responses'] = self.instance.get_responses.return_value
+
+        self.instance.data_api_client.get_brief.return_value = brief
+
+        with po(buyers, 'get_framework_and_lot') as get_framework_and_lot:
+            with po(buyers, 'is_brief_correct') as is_brief_correct:
+                with mock.patch.object(buyers, 'current_user') as current_user:
+                    result = self.instance.get_context_data(**kwargs)
+
+        is_brief_correct.assert_called_once_with(brief['briefs'],
+                                                 kwargs['framework_slug'],
+                                                 kwargs['lot_slug'],
+                                                 current_user.id)
+
+        self.instance.data_api_client.get_brief\
+            .assert_called_once_with(kwargs['brief_id'])
+
+        self.instance.get_responses\
+            .assert_called_once_with(brief['briefs'])
+
+        assert result == expected
+
+    def test_get_context_data_incorrect_brief(self):
```

Comment:
Super small but naming your test based on the situation + expected result would make this more readable e.g. `test_get_context_data_with_incorrect_brief_will_abort`, same as below with `test_get_context_data_with_open_brief_will_abort`.